components/eam/src/physics/cam/physpkg.F90

```diff
@@ -1196,9 +1205,14 @@ subroutine phys_run1_adiabatic_or_ideal(ztodt, phys_state, phys_tend,  pbuf2d)
 
        ! Dump dynamics variables to history buffers
        call diag_phys_writeout(phys_state(c))
+
+       ! --JH--: Allow advecting of CLDERA passive tendencies if enabled
```

Comment:
I don't feel strongly enough to ask for a commit to change, and there's plenty of folks on E3SM who follow this as well, but I don't particularly like adding initials/signatures to code contributions within the code. I think it just ends up detracting from the comments in the code rather than bringing value. All this info will be in the git history regardless. But, again, I don't feel strongly enough to request a change and plenty of others do this as well.

Reply:
Fair enough.  I do this mostly for dev purposes e.g. searching my initials to jump between relevant sections, so it probably should have been removed before a PR.